src/darwin/Framework/CHIPTests/MTROTAProviderTests.m

```diff
@@ -634,7 +664,67 @@ - (XCTestExpectation *)announceProviderToDevice:(MTRDevice *)device
     return responseExpectation;
 }
 
-- (void)test001_ReceiveOTAQuery
+/**
+ * Helper that checks if the OTA provider locations match.
+ */
+- (bool)AreEqual:(MTROtaSoftwareUpdateRequestorClusterProviderLocation *)pl1
+    providerLocation:(MTROtaSoftwareUpdateRequestorClusterProviderLocation *)pl2
+{
+    return [pl1.providerNodeID isEqualToNumber:pl2.providerNodeID] && [pl1.endpoint isEqualToNumber:pl2.endpoint] &&
+        [pl1.fabricIndex isEqualToNumber:pl2.fabricIndex];
+}
+
+- (void)test001_UpdateDefaultOTAProviderAndVerify
```

Comment:
Sunk cost since you have already done all the renaming, but: this should just have been added at the end, no?  In general, we want our tests to be able to run in any order.  We are not there yet, but we are working on it....

Reply:
yeah this test is unrelated to any others. its just tests the write attribute api for default provider. Removed this anyways.